ecl/eclcc/CMakeLists.txt

```diff
@@ -61,17 +62,18 @@ endif()
 HPCC_ADD_EXECUTABLE ( eclcc ${SRCS} )
 install ( TARGETS eclcc RUNTIME DESTINATION ${EXEC_DIR} )
 target_link_libraries ( eclcc 
-         jlib
-         nbcd 
          eclrtl 
          deftype 
          dafsclient 
          dalibase 
-         workunit 
-         thorhelper 
+         dllserver
          hql
          hqlcpp
-         dllserver
+         jlib
+         nbcd 
+         thorhelper 
+         workunit 
+         ws_dfsclient
```

Comment:
@richardkchapman I'm a bit concerned about this.  It means eclcc now has dependencies on esphttp, xmllib.  I'm not sure what implications it has for the client tools.  Possibly none since the ecl command already pulls them in.

I would be very tempted to get rid of the code that looks up the current record definition in dali, or at least mark it as deprecated.  The record translation is a much better bet.

Reply:
I'm not sure how the dali record layout is used by the compiler...
It generates static transform at compiler time? What controls when it does that currently?